tests/render/test_render_BulletListContentBlock.py

```diff
@@ -69,75 +64,18 @@ def test_parse_row_condition_string_pandas_engine():
-    pattern = os.path.join(  # noqa: PTH118 # FIXME CoP
-        dir_path, "..", "..", "tests", "test_definitions", "*", "expect*.json"
-    )
-    test_files = glob.glob(pattern)  # noqa: PTH207 # FIXME CoP
-
-    # Historically, collecting all the JSON tests was an issue - this step ensures we actually have test data.  # noqa: E501 # FIXME CoP
-    assert len(test_files) == 61, (
-        "Something went wrong when collecting JSON Expectation test fixtures"
-    )
-
-    # Not ported over to V1 so ignore for purposes of this test
-    UNSUPPORTED_EXPECTATIONS = {
-        "expect_column_pair_cramers_phi_value_to_be_less_than",
-        "expect_column_parameterized_distribution_ks_test_p_value_to_be_greater_than",
-        "expect_column_chisquare_test_p_value_to_be_greater_than",
-    }
-
-    # Loop over all test_files, datasets, and tests:
-    test_results = defaultdict(list)
-    for filename in test_files:
-        test_definitions = json.load(open(filename))
-        if test_definitions["expectation_type"] in UNSUPPORTED_EXPECTATIONS:
-            continue
-        for dataset in test_definitions["datasets"]:
-            for test in dataset["tests"]:
-                # Construct an expectation from the test.
-                if type(test["in"]) == dict:  # noqa: E721 # FIXME CoP
-                    # Skip tests with invalid configurations
-                    if test["in"].get("catch_exceptions"):
-                        continue
-                    fake_expectation = ExpectationConfiguration(
-                        type=test_definitions["expectation_type"],
-                        kwargs=test["in"],
-                    )
-                else:
-                    # This would be a good place to put a kwarg-to-arg converter
-                    continue
-
-                # Attempt to render it
-                render_result = ExpectationSuiteBulletListContentBlockRenderer.render(
-                    [fake_expectation]
-                )
-                assert render_result is not None
-                render_result = render_result.to_json_dict()
-
-                assert isinstance(render_result, dict)
-                assert "content_block_type" in render_result
-                assert render_result["content_block_type"] in render_result
-                assert isinstance(render_result[render_result["content_block_type"]], list)
-
-                # TODO: Assert that the template is renderable, with all the right arguments, etc.
-                # rendered_template = pTemplate(el["template"]).substitute(el["params"])
-
-                test_results[test_definitions["expectation_type"]].append(
-                    {
-                        test["title"]: render_result,
-                        # "rendered_template":rendered_template
-                    }
-                )
-
-    # TODO: accommodate case where multiple datasets exist within one expectation test definition
-    with open(
```

Comment:
I've removed this since it doesn't seem to be testing anything. Maybe it tests that the results are json serializable? Every result should be json serializable because we have called `to_json_dict` above and unit tests for that behavior should go there. Also, we don't need to actually write a file to verify this.